.github/workflows/test_docker-build.yml

```diff
@@ -0,0 +1,116 @@
+name: Test All Docker Images Build
```

Comment:
If this is a test for all Docker images in Weaver, we should include the relay images too, no?